README.md

```diff
@@ -86,6 +86,32 @@ Pirk employs generic [data and query schemas](http://pirk.incubator.apache.org/f
 	    See the License for the specific language governing permissions and
 	    limitations under the License. See accompanying LICENSE file.
 
+## Export Control
+
+This distribution includes cryptographic software. The country in which you 
+currently reside may have restrictions on the import, possession, use, and/or
+re-export to another country, of encryption software. BEFORE using any 
+encryption software, please check your country's laws, regulations and 
+policies concerning the import, possession, or use, and re-export of encryption
+software, to see if this is permitted. See <http://www.wassenaar.org/> for more
+information.
+
+The U.S. Government Department of Commerce, Bureau of Industry and Security 
+(BIS), has classified this software as Export Commodity Control Number (ECCN) 
+5D002.C.1, which includes information security software using or performing 
+cryptographic functions with asymmetric algorithms. The form and manner of this
+Apache Software Foundation distribution makes it eligible for export under the 
+License Exception ENC Technology Software Unrestricted (TSU) exception (see the
+BIS Export Administration Regulations, Section 740.13) for both object code and
+source code.
+
+The following provides more details on the included cryptographic software: 
+
+Apache Pirk implements cryptographic software and is designed for use with the
```

Comment:
I believe this to be sufficient.  I have avoided describing precisely [the nature of the implemented cryptography](http://www.apache.org/dev/crypto.html#classify) as (a) it may well change, and we'd have to keep the various docs up to date, and (b) if we mis-classify, that may be worse than being vague. But others may well feel differently...